openedx/core/djangoapps/oauth_dispatch/toggles.py

```diff
@@ -0,0 +1,11 @@
+"""
+Feature toggle code for oauth_dispatch.
+"""
+
+from edx_rest_framework_extensions.config import SWITCH_ENFORCE_JWT_SCOPES
+
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace, WaffleSwitch
+
+WAFFLE_NAMESPACE = 'oauth2'
+OAUTH2_SWITCHES = WaffleSwitchNamespace(name=WAFFLE_NAMESPACE)
+ENFORCE_JWT_SCOPES = WaffleSwitch(OAUTH2_SWITCHES, SWITCH_ENFORCE_JWT_SCOPES)
```

Comment:
Note: As of now, this will put the `oauth2` namespace twice since `SWITCH_ENFORCE_JWT_SCOPES` already includes the namespace.  However, my latest PR will change this: https://github.com/edx/edx-drf-extensions/pull/36/files#diff-7a662b4423d50ddd106d86b4670637aeR6